Training/interface/DataLoader_main.h

```diff
@@ -0,0 +1,770 @@
+#include <boost/preprocessor/variadic.hpp>
+#include <boost/math/constants/constants.hpp>
+
+#include "TauMLTools/Core/interface/AnalysisMath.h"
+#include "TauMLTools/Core/interface/RootExt.h"
+#include "TauMLTools/Analysis/interface/TauTuple.h"
+
+#include "TROOT.h"
+#include "TLorentzVector.h"
+#include "DataLoader_setup.h"
```

Comment:
It will be more flexible if the setup is not included directly in `DataLoader_main.h`. Like this, we'll have a possibility to have different setups without changing the DataLoader code.